MySqlFiles/HighSchool.sql

```diff
@@ -0,0 +1,47 @@
+CREATE DATABASE high_school;
+
+CREATE TABLE IF NOT EXISTS student(
+	registration_number int NOT NULL PRIMARY KEY,
+    dni_number char(8) UNIQUE NOT NULL,
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,	
+    birth_date date
+);
+
+CREATE TABLE IF NOT EXISTS teacher(
+	teacher_id int NOT NULL PRIMARY KEY,
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,	
+    birth_date date
+);
+
+CREATE TABLE IF NOT EXISTS course(
+	course_id int NOT NULL PRIMARY KEY,
+    course_name varchar(255) NOT NULL,
+    assigned_teacher varchar(255),
```

Comment:
The assigned teacher should reference the teachers table primary key, that is an int